packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -93,6 +93,9 @@ export class DocumentService implements api.IDocumentService {
      * @returns returns the document delta storage service for routerlicious driver.
      */
     public async connectToDeltaStorage(): Promise<api.IDocumentDeltaStorageService> {
+        if (this.documentStorageService === undefined) {
+            await this.connectToStorage();
```

Comment:
Consider making this call unconditional, and make connectToStorage() itself return same object when called multiple times.
Basically - is there any value in creating multiple storage objects? If the answer is no, then we should not do so.

Reply:
`connectToDeltaStorage` always calls `connectToStorage` now. However, `connectToStorage` will immediately return the `documentStorageService` object if its already available